test/src/Trait/TestFunctionalTrait.php

```diff
@@ -196,11 +195,16 @@ protected function assertQueryResultContainsString(string $query, string $string
 	 * @return array<string>
 	 * @throws Exception
 	 */
-	protected static function runSqlQuery(string $query, bool $redirectOutput = true): array {
```

Comment:
We should fix conflicts because as I remember this method also was modified to support \G line ending